devdocs.el

```diff
@@ -94,6 +94,11 @@ Fontification is done using the `org-src' library, which see."
   "Whether to select the DevDocs window for viewing."
   :type 'boolean)
 
+(defcustom devdocs-extra-rendering-functions '()
```

Comment:
Let's make this just a defvar, since there's no reasonable setting the user can do via `M-x customize`.